client/post-editor/style.scss

```diff
@@ -10,6 +10,12 @@
 	z-index: z-index( 'root', '.is-group-editor::before' );
 }
 
+@include breakpoint( "<660px" ) {
+	.is-group-editor.focus-sidebar::before {
```

Comment:
The `background-color` of the entire screen is set using this layer (see the first lines of this file). For mobile viewports, the editor background color must be white (everything fine), but for the sidebar it must be gray, that's what this media query is for. I couldn't find another reasonable way of doing that, because all the `<div>` in the hierarchy stretch to fit the content, not even the `<body>` is set to expand to the whole screen.